cypress/integration/vendor-cwu-UD.spec.js

```diff
@@ -0,0 +1,96 @@
+/// <reference types="cypress" />
+
+describe('As a user authenticated via GitHub,',  function() {
+    beforeEach(function() {
+        cy.sqlFixture('dbReset.sql')
+        cy.login('vendor')
+        cy.sqlFixture('cwuOpportunity.sql')
+        cy.sqlFixture('cwuProposal.sql')
+    })
+
+    it('edit CWU proposal', function() {
+        cy.get('#user-sign-up-step-two-terms').click()
+        cy.get('a').contains('Complete Profile').click()
+        cy.visit('/opportunities')
```

Comment:
There's a button you can click, but I was having a lot of timeout issues around it, so I did this instead and it seemed (?) to help